Curs/src/Data/xhrNew.json

```diff
@@ -0,0 +1,530 @@
+{
+    "selectorContent": [
```

Comment:
Not sure whether you're trying to abstract away common configuration options with the `selectorContent` array here, but this four-value array seems to be all over the place in this file. While it is true that this _is_ indeed the more flexible option, oftentimes we have to find the right balance between future-proofing, extensibility, ease of maintenance and making sense. :)

In our particular case, I'd jump at the chance to clean up this config file to only list out these repeated arrays once. Keeping things DRY is almost always a (very!) good idea.

Reply:
You mean that I have to remove the selectorContent array  just from line 2 right?

Reply:
Because I need them just as options for my selectors.